### PR TITLE
TF-726: add text_color background_color text_font setting

### PR DESCRIPTION
**Pull request by ManhNTX**
https://user-images.githubusercontent.com/107173849/180655242-b3edeef9-81cf-47ff-9aab-592110b3880b.mov

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -124,7 +124,7 @@ dependencies:
   # enough_html_editor: Compose email for Mobile & Tablet
   enough_html_editor:
     git:
-      url: https://github.com/dab246/enough_html_editor.git
```
Comment (hoangdat) [inline pubspec.yaml:127]: should move to Linagora repository?

**Comment by hoangdat** [comment]
please update with new design at: https://www.figma.com/file/rRVxJpCu1G22D0jqyvK7I9/TeamMail?node-id=4501%3A190254

File: pubspec.yaml (hunk 1 above)
Comment (ManhNTX) [reply pubspec.yaml:127]: After finish all richtext function I will remove it

Comment (hoangdat) [reply pubspec.yaml:127]: okie #753

**Review by dab246 — APPROVED** [review]
/lgtm

**Pull request merged**